README.md

```diff
@@ -1,149 +1,277 @@
-    #[Column(type: 'primary', primary: true)]
-    protected int $id;
+    // ...
 
-    #[Column(type: 'string')]
-    protected string $name;
+    #[HasMany(target: Post::class)]
+    private array $posts;
 }
+```
+
+> **Note**
+> Read more about [HasMany](https://cycle-orm.dev/docs/relation-has-many).
+
+#### BelongsTo
+
+```php
+use Cycle\Annotated\Annotation\Entity;
+use Cycle\Annotated\Annotation\Relation\BelongsTo;
 
 #[Entity]
-#[InheritanceSingleTable]
-class Employee extends Person
+class Post
 {
-    #[Column(type: 'int')]
-    protected int $salary;
+    // ...
+
+    #[BelongsTo(target: User::class)]
+    private User $user;
 }
+```
+
+> **Note**
+> Read more about [BelongsTo](https://cycle-orm.dev/docs/relation-belongs-to).
+
+#### RefersTo
+
+```php
+use Cycle\Annotated\Annotation\Entity;
+use Cycle\Annotated\Annotation\Relation\RefersTo;
+use Cycle\Annotated\Annotation\Relation\HasMany;
 
 #[Entity]
-#[InheritanceSingleTable(value: 'foo_customer')]
-class Customer extends Person
+class User
 {
-    #[Column(type: 'json')]
-    protected array $preferences;
+    // ...
+
+    #[RefersTo(target: Comment::class)]
+    private ?Comment $lastComment;
+
+    #[HasMany(target: Comment::class)]
+    public array $comments;
+
+    // ...
+
+    public function addComment(Comment $c): void
+    {
+        $this->lastComment = $c;
+        $this->comments[] = $c;
+    }
+    
+    public function removeLastComment(): void
+    {
+        $this->lastComment = null;
+    }
+    
+    public function getLastComment(): ?Comment
+    {
+        return $this->lastComment;
+    }
 }
 ```
 
-#### Joined Table Inheritance
+> **Note**
+> Read more about [RefersTo](https://cycle-orm.dev/docs/relation-refers-to).
+
+#### ManyToMany
 
 ```php
+use Cycle\Annotated\Annotation\Relation\ManyToMany;
+use Cycle\Annotated\Annotation\Entity;
+
 #[Entity]
-class Person
+class User
 {
-    #[Column(primary: true)]
-    protected int $id;
+    // ...
+
+    #[ManyToMany(target: Tag::class, through: UserTag::class)]
+    protected array $tags;
     
-    #[Column()]
-    protected int $fooId;
+    public function getTags(): array
+    {
+        return $this->tags;
+    }
+    
+    public function addTag(Tag $tag): void
+    {
+        $this->tags[] = $tag;
+    }
+    
+    public function removeTag(Tag $tag): void
+    {
+        $this->tags = array_filter($this->tags, static fn(Tag $t) => $t !== $tag);
+    }
+}
+```
+
+> **Note**
+> Read more about [ManyToMany](https://cycle-orm.dev/docs/relation-many-to-many).
+
+#### Embedded Entities
+
+```php
+use Cycle\Annotated\Annotation\Embeddable;
+use Cycle\Annotated\Annotation\Column;
+
+#[Embeddable]
+class UserCredentials
+{
+    #[Column(type: 'string(255)')]
+    public string $username;
 
     #[Column(type: 'string')]
-    protected string $name;
+    public string $password;
 }
 
+use Cycle\Annotated\Annotation\Entity;
+use Cycle\Annotated\Annotation\Column;
+use Cycle\Annotated\Annotation\Relation\Embedded;
```

Comment:
Merge it with the previous uses block please

Reply:
Done.